### DISCUSSION TITLE
Table distance

### DISCUSSION THREAD

**Discussion by Syab05** [acorg/Racmacs#166] [Q&A]
Hi! 

I am trying to get the antigenic distance from my data, as I read at https://acorg.github.io/Racmacs/articles/intro-to-antigenic-cartography.html , the examples show a distance table example, in which every homologous Ag -Sr is 0. 
However, when I tried with my data, some homologue data was not equal to zero. Does this kind of result usually occurs?
I did tabledistances() to acquired the data.

Kind regards,
Syabilla

**Comment by drserajames** [comment]
Hi Syabilla!

It's normal to have some distance between homologous antigens and sera. This is because another serum has a higher titres to that antigen (which can either be a true result or due to measurement error). It is generally okay, but can sometimes cause problems e.g some antigens have higher titres to all sera, which can distort the map.

**Reply by Syab05** [reply]
Thank you Sarah your answer is very helpful. I have a follow-up question.
After generating the map, I am succesfully change the color of the antigen of my interests. However, I would like for the label to appear, would you mind suggesting how to do that?

Kind regards

**Reply by drserajames** [reply]
Have you tried the `plot_labels = TRUE` in the plot function?

From the help file:

```
plot(
  x,
  optimization_number = 1,
  xlim = NULL,
  ylim = NULL,
  plot_ags = TRUE,
  plot_sr = TRUE,
  plot_labels = FALSE,
  plot_blobs = TRUE,
  point_opacity = "automatic",
  show_procrustes = TRUE,
  show_error_lines = FALSE,
  plot_stress = FALSE,
  indicate_outliers = "arrowheads",
  grid.col = "grey90",
  grid.margin.col = "grey50",
  outlier.arrow.col = grid.col,
  fill.alpha = 0.8,
  outline.alpha = 0.8,
  procrustes.lwd = 2,
  procrustes.col = "black",
  procrustes.arr.type = "triangle",
  procrustes.arr.length = 0.2,
  procrustes.arr.width = 0.15,
  label.offset = 0,
  padding = 1,
  cex = 1,
  margins = rep(0.5, 4),
  ...
)
```

`plot_labels`
should point labels be plotted, can be true, false or "antigens" or "sera"

**Reply by shwilks** [reply]
You can also take a look at this Gist showing how you can add smarter labels to a map

https://gist.github.com/shwilks/6b3d1fe98005b492de317a1ac3284fb0
